kobuki_gazebo_plugins/src/gazebo_ros_kobuki_loads.cpp

```diff
@@ -309,6 +309,8 @@ void GazeboRosKobuki::setupRosApi(std::string& model_name)
 {
   std::string base_prefix;
   gazebo_ros_->node()->param("base_prefix", base_prefix, std::string("mobile_base"));
```

Comment:
I believe the default value `std::string("mobile_base")` has to be dropped. Otherwise you won't be able to set the `base_prefix` to be an empty string.